ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_linalg/test_matrix_and_vector_products.py

```diff
@@ -0,0 +1,40 @@
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_float_dtypes,
+        min_num_dims=3,
+        max_num_dims=5,
+        min_dim_size=2,
+        max_dim_size=5,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.linalg.matmul"
+    ),
+    fm=st.booleans(),
+)
+
+def test_matmul(
+    x1: array,
+    x2: array,
+    /,
+    out=None,
+    *,
+    casting='same_kind',
+    order='K',
+    dtype=None,
+    subok=True
+):
+    helpers.test_frontend_function(
+        input_dtypes=[dtype],
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="numpy",
+        fn_tree="linalg.matmul",
+        rtol=1e-02,
+        a=np.array(x, dtype=dtype),

```

Comment:
matmul takes 2 inputs, so please add the arguments here as well. Please refer [example](https://github.com/unifyai/ivy/blob/6f27c45a4deef5ec2ad1e5117ef2937ce6f99943/ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py#L204) and do the changes accordingly